torch/onnx/symbolic_opset12.py

```diff
@@ -17,9 +17,9 @@ def einsum(g, equation, tensor_list):
 
 @parse_args('v', 'f', 'i')
 def dropout(g, input, p, train):
-    sym_help.assert_training_mode(train, "dropout")
+    sym_help.check_training_mode(train, "dropout")
     # in eval mode, dropout is non-op - if the node's train param is set to False, dropout is non-op
```

Comment:
update comment?

Reply:
Do you mean the comment in line 21? I think this is not changed, since it is pointing to the training param for this node (op).